dictionary-octopus.txt

```diff
@@ -12,6 +12,7 @@ approot
 appsettings
 Appstep
 Appsteps
+argparse
```

Comment:
These were needed to be added as they are in the code samples and can't be changed